zenoblend/BlenderMesh.h

```diff
@@ -34,4 +34,8 @@ using BlenderInputNamesType = std::set<std::string>;
 using BlenderInputsType = std::map<std::string, std::function<std::shared_ptr<zeno::BlenderAxis>()>>;
 using BlenderOutputsType = std::map<std::string, std::shared_ptr<zeno::BlenderAxis>>;
 
+using LineViewerVertexBufferType = std::vector<std::vector<float>>;
```

Comment:
Why not `std::vector<zeno::vec3f>`?

Reply:
If you worry zeno::vec3f can't be passed to python, may use `std::vector<std::array<float, 3>>` for better performance.

Reply:
I used `std:vector` instead of `std::array` just because `pybind11/stl_bind.h` only offers method binding vector and map containers and there is no function named `py::bind_array` available for making opaque types. As the [document](https://pybind11.readthedocs.io/en/stable/advanced/cast/stl.html#making-opaque-types) suggests, it is possible to bind it manually, but unfortunately I haven't ever found any information about it in Internet. I believe I can do some experiments first to figure out whether it can be easily done.